test/fixtures/pretty-print.json

```diff
@@ -1,14 +1,14 @@
 {
   "chalk": {
-    "replace": true,
+    "npm": true,
```

Comment:
`replace:true` does nothing now so I figured that I would just update the code to use `npm:true`